src/common/resources/voucher.js

```diff
@@ -17,6 +17,14 @@ export const VOUCHER = {
   },
 };
 
+export const VOUCHER_ID = {
```

Comment:
why id?

Reply:
We have another resource named `VOUCHER`. Here I fetch a voucher by id

Reply:
`VOUCHER_BY_ID` ?